java/runtime/src/main/java/org/ray/runtime/config/RayConfig.java

```diff
@@ -196,6 +201,9 @@ public RayConfig(Config config) {
       driverResourcePath = null;
     }
 
+    // number of threads that execute tasks under SINGLE_PROCESS mode
```

Comment:
```suggestion
    // Number of threads that execute tasks under SINGLE_PROCESS mode.
```
Also apply to other places.